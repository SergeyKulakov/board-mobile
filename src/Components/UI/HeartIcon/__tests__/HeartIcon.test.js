import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import { HeartIcon as Component } from '../index'

describe('HeartIcon', () => {
  const defaultProps = {
    isActive: true,
    size: 12,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    const withoutActive = shallow(
      <Component {...defaultProps} isActive={false} />,
    )

    expect([wrapper, withoutActive]).toMatchSnapshot()
  })
})
