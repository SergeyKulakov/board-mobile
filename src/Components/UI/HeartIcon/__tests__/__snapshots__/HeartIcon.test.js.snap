// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeartIcon render snapshot 1`] = `
Array [
  <Icon
    color="#ff0000"
    name="heart"
    size={12}
    visible={true}
  />,
  <Icon
    color="#fff"
    name="heart-o"
    size={12}
    visible={true}
  />,
]
`;
